packages/trpc/server/routers/viewer/eventTypes/getByViewer.handler.ts

```diff
@@ -172,13 +172,19 @@ export const getByViewerHandler = async ({ ctx }: GetByViewerOptions) => {
   const mergedEventTypes = Object.values(eventTypesHashMap)
     .map((eventType) => eventType)
     .filter((evType) => evType.schedulingType !== SchedulingType.MANAGED);
+
+  let image = user.avatar || undefined
+  if (image && user?.username) {
+    image = `${CAL_URL}/${user.username}/avatar.png`
+  }
```

Comment:
We can simplify this
```suggestion
  const image = user?.username ? `${CAL_URL}/${user.username}/avatar.png` : undefined;
```

Reply:
@roae Its not guaranteed users will always have avatar, no? Again we could add more condition but code will become messy

Reply:
@asadath1395 The avatar.png endpoint show a placeholder if the user hasn’t avatar